examples/hello_flutter/BUILD.gn

```diff
@@ -4,7 +4,7 @@
 
 import("//flutter/build/flutter_app.gni")
 
-flutter_app("hello_flutter") {
+flutter_jit_app("hello_flutter") {
```

Comment:
I think the idea here is to build the hello_flutter app both ways to make sure that both ways work.